README.md

```diff
@@ -1 +1,31 @@
-# animal-shelter
\ No newline at end of file
+# Animal shelter
+
+## Motivation
+
+The idea of this project is to help with understanding and consolidate knowledge about different technologies used in programming. 
+
+## About project
+
+### Aim
+The project was created to learn and practice using these skills in a real platform with real problems. 
+
+The `Animal shelter` is an application that focuses on simple operations in the beginning and in process of time will be developed and new functionalities will be added. 
+
+### The main goal
+The `Animal shelter` stores data about animals and employees (zookeepers, doctors etc). Both animals and employees can be added, delete, update, and read data. 
+
+### Technologies
+In this project the following technologies will be used:
+*Java 11, Spring, H2 database (in the beginning), Mockito, Junit.*
```

Comment:
Also add Maven, Hibernate(I think that we will use it) here.

Reply:
Thanks, done.